### PR TITLE
Issue #2029: [UX] Remove ordering/weight from the vocabularies listing page

### LINKED ISSUE #2029: [UX] Why are taxonomies orderable?
Is this left over from Drupal 6 when they all used to be thrown into the same form element? Does this order / value do anything on Backdrop sites? If not, can we please kill the tabledrag on this page?  It's confusing to users.
screen shot 2016-07-26 at 11 25 17 am

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/2029

Replaces https://github.com/backdrop/backdrop/pull/2736

### CODE REVIEW

**Comment by klonos** [comment]
...too many changes in subsequent commits. Refreshing the sandbox.

**Comment by docwilmot** [comment]
The sandbox is broken, can we reboot please?

File: core/modules/taxonomy/taxonomy.module [hunk 1]
```diff
@@ -272,8 +272,14 @@ function taxonomy_theme() {
   );
 
   return array(
-    'taxonomy_overview_vocabularies' => array(
-      'render element' => 'form',
+    'taxonomy_vocabulary_name' => array(
+      'variables' => array('vocabulary' => NULL),
+    ) + $base,
+    'taxonomy_vocabulary_description' => array(
```
Comment (docwilmot) [inline core/modules/taxonomy/taxonomy.module:278]: Do these really need to be fully themed? Theyre just plain text and integer.

File: core/modules/taxonomy/taxonomy.module [hunk 2]
```diff
@@ -272,8 +272,14 @@ function taxonomy_theme() {
...
+      'variables' => array('vocabulary' => NULL),
+    ) + $base,
+    'taxonomy_vocabulary_term_count' => array(
```
Comment (docwilmot) [inline core/modules/taxonomy/taxonomy.module:281]: Do these really need to be fully themed? Theyre just plain text and integer.

**Comment by jenlampton** [comment]
Closing for new sandbox.

**Comment by jenlampton** [comment]
Sandboxme!

**Comment by jenlampton** [comment]
looks like a new sandbox didn't work...  Replaced by https://github.com/backdrop/backdrop/pull/3127

**Pull request closed without merging**